waku/v2/protocol/waku_swap.nim

```diff
@@ -76,8 +76,33 @@ proc init*(T: type Cheque, buffer: seq[byte]): ProtoResult[T] =
 # Accounting
 #
 
-proc accountFor*(peerId: PeerId, n: int) {.gcsafe.} =
-  info "Accounting for", peerId, n
+proc init*(wakuSwap: WakuSwap) =
+  info "wakuSwap init 1"
+  proc handle(conn: Connection, proto: string) {.async, gcsafe, closure.} =
+    info "NYI swap handle incoming connection"
+
+  proc accountFor(peerId: PeerId, n: int) {.gcsafe, closure.} =
+    info "Accounting for", peerId, n
+    info "Accounting test", text = wakuSwap.text
+    # Nicer way to write this?
+    if wakuSwap.accounting.hasKey(peerId):
+      wakuSwap.accounting[peerId] += n
+    else:
+      wakuSwap.accounting[peerId] = n
```

Comment:
```suggestion
      wakuSwap.accounting[peerId] += n
```

@oskarth I feel like just this should work, but I may be wrong.

Reply:
Nope, that is a compile error when I tried it IIRC

Reply:
Or key not found error at runtime, forgot

Reply:
Ah sad. Then idk, other than that this looks fine to me